### PR TITLE
Aus content updates

### PR DESCRIPTION
**Pull request by mkllnk**
#### What? Why?

The Australian team updated their pricing model which needed some content changes on the website. Since we are using the source locale `en.yml` we need to change text via pull requests instead of going through Transifex.

We also wanted to change the alert banner on the home page. I guessed that this customisation could be desirable for other instances and made it customisable via the admin interface. It's still 100% downwards compatible and the usual register call is shown if you don't put a custom text in the admin panel. While the usual register call is translated into different languages, any custom text used for the alert banner would be static and not translated. So until we get to the multi-lang 2.0 feature, the custom banner may not be attractive to multi-lang instances.

#### What should we test?

- The alert banner and menu items look as usual.
- The alert banner can be customised via Admin / Configuration / Content.

This is on Australian production already. A quick test on staging that it doesn't mess up anything else would still be nice. :-)

#### Release notes

Added: The homepage alert banner can be customised by admin users.

Changelog Category: Added

### CODE REVIEW

File: app/models/content_configuration.rb [hunk 1]
```diff
@@ -12,6 +12,20 @@ class ContentConfiguration < Spree::Preferences::FileConfiguration
...
+  # I intended to set a default for home_page_alert_html including I18n text.
+  #
+  #      default: "<a href='/register' target='_blank'>
+  #             #{I18n.t('shared.register_call.selling_on_ofn')}&nbsp;
+  #             <strong>#{I18n.t('shared.register_call.register')}
+  #             <i class='ofn-i_054-point-right'></i></strong>
+  #             </a>"
+  #
+  # Unfortunately, this configuration is initialised and cached before locales
+  # are loaded. So I chose to not set a default and use a translatable view instead.
```
Comment (luisramos0) [inline app/models/content_configuration.rb]: I dont think we should tell a story in comments. imo the story can go in the commit message or the PR. comments should be straight to the point about the current state of the code. I'd keep only these 2 last lines:
# All the following defaults using I18n don't work.
# https://github.com/openfoodfoundation/openfoodnetwork/issues/3816

**Comment by sigmundpetersen** [comment]
Yes, `en_AU.yml`. Then en.yml could be the source file that is actually not used on any instance. I think that would be good.

File: app/models/content_configuration.rb (hunk 1 above)
Comment (mkllnk) [reply app/models/content_configuration.rb]: You are right, I covered that in the issue now. There was a point in my development when that comment made more sense and I didn't want to lose the HTML code to paste. But in the end I kept the view and the default is not need. :+1:

**Comment by lin-d-hop** [comment]
Tests look good!
Testing notes here:
https://docs.google.com/document/d/1UUm4MqJbktMiaOcWCSsrAuRpq5ZXDpZ6brxduSKMj3Y/edit#

**Pull request merged**